src/prop/prop_proof_manager.h

```diff
@@ -270,6 +278,12 @@ class PropPfManager : protected EnvObj
   context::CDHashSet<Node> d_inputClauses;
   /** Asserted clauses derived from lemmas */
   context::CDHashSet<Node> d_lemmaClauses;
+  /** Are we debugging inference identifiers? */
+  bool d_debugLemmaClauseIds;
```

Comment:
Maybe we could call this something else? "Debugging" gives me idea of something internal, not something that we may give as output to the user.